lte/gateway/c/session_manager/LocalSessionManagerHandler.cpp

```diff
@@ -47,11 +47,21 @@ void LocalSessionManagerHandlerImpl::ReportRuleStats(
     PrintGrpcMessage(
         static_cast<const google::protobuf::Message&>(request_cpy));
   }
-  MLOG(MDEBUG) << "Aggregating " << request_cpy.records_size() << " records";
   enforcer_->get_event_base().runInEventBaseThread([this, request_cpy]() {
+    if (!session_store_.is_ready()) {
+      // Since PipelineD reports a delta value for usage, this could lead to
+      // SessionD missing some usage if Redis becomes. However, since we usually
+      // only see this case on service restarts, we'll let this slide for now.
+      // Once we move to flat-rate reporting from PipelineD this will no longer
+      // be an issue.
```

Comment:
so for now we will give free data while redis is not available, right?

Reply:
Yes.